src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,84 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  currentPage: number;
+  totalPages: number;
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  totalPages,
+  perPage,
+  onPageChange,
+}) => {
+  const handleArrowClick = (direction: string, condition: boolean) => {
```

Comment:
Instead of using a string for the direction parameter, consider using a string literal type to restrict the possible values to 'prev' and 'next'.